lib/skywalking/util.lua

```diff
@@ -15,6 +15,16 @@
 -- limitations under the License.
 --
 
+-- source: https://github.com/apache/apisix/blob/master/apisix/core/string.lua#L18
+local type = type
+local ffi         = require("ffi")
```

Comment:
Where is `ffi` from? I can't see the dependency change.

Reply:
http://luajit.org/ext_ffi.html

Reply:
But you never declare it, and CI fails.

Reply:
@membphis How should this lib be managed? Is it native in OpenRestly's Lua engine already?

Reply:
> @membphis How should this lib be managed? Is it native in OpenRestly's Lua engine already?

i saw Apisix not declare too. i think native in OpenRestly.

Reply:
> i think native in OpenRestly.

yes, you are right.